src/examples/wasm/main.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
Perhaps it would be cleaner to name this file server.go? That makes it more obvious what it is, and that it should be compiled with `go` and not `tinygo`.

Reply:
Fixed